addon/styles/_frost-tab.scss

```diff
@@ -0,0 +1,15 @@
+.frost-tab {
+  padding-bottom: 2px;
+  border-bottom: 5px solid transparent;
+  cursor: default;
+
+  .tab {
+    &.active {
+      border-bottom: 5px solid $frost-color-blue-1;
```

Comment:
New convention I'm trying to push, can you move the variables to aliases at the top of the file?

e.g.

```
$frost-tab-active-border: $frost-color-blue-1;

.frost-tab .tab.active {
  border-bottom: 5px solid $frost-tab-active-border;
}
```